test/unit/org/apache/cassandra/service/accord/AccordStateCacheTest.java

```diff
@@ -428,7 +428,7 @@ public void evictionBlockedOnSaving()
             instance.release(item);
         }
 
-        assertCacheState(cache, 0, 4, nodeSize(1) * 3 + nodeSize(3));
```

Comment:
these tests have been failing for so long... finally got around to them.

the string is `0*` which is size 2... not 3...